march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -255,19 +268,31 @@ void MarchHardwareInterface::read(ros::Duration elapsed_time)
 
 void MarchHardwareInterface::write(ros::Duration elapsed_time)
 {
-  positionJointSoftLimitsInterface.enforceLimits(elapsed_time);
-
   for (int i = 0; i < num_joints_; i++)
   {
     march4cpp::Joint joint = marchRobot.getJoint(joint_names_[i]);
 
+    // Enlarge joint_effort_command so dynamic reconfigure can be used inside it's bounds
```

Comment:
Only when actuationmode = torque?

Reply:
I had to set this before the limits were enforced. Because last time the torque mode was reviewed, Isha requested that both limits had to be enforced. I could both move inside the if loop.